x-pack/plugins/ml/public/application/overview/components/anomaly_detection_panel/table.tsx

```diff
@@ -169,17 +173,17 @@ export const AnomalyDetectionTable: FC<Props> = ({ items, jobsList, statsBarData
     },
   ];
 
-  const onTableChange = ({
+  const onTableChange: EuiInMemoryTable<Group>['onTableChange'] = ({
     page = { index: 0, size: 10 },
-    sort = { field: AnomalyDetectionListColumns.id, direction: SORT_DIRECTION.ASC },
-  }: OnTableChangeArg) => {
+    sort = { field: AnomalyDetectionListColumns.id, direction: 'asc' },
```

Comment:
Sorting on the anomaly score column doesn't seem to work properly anymore - I can't get the descending order to work (this should be sorted in descending order):

![image](https://user-images.githubusercontent.com/7405507/80959113-08f98000-8dfe-11ea-87a7-15df4d87c35f.png)

compared to master:

![image](https://user-images.githubusercontent.com/7405507/80959230-465e0d80-8dfe-11ea-855e-0bad8bd97c63.png)

Reply:
I can reproduce it but I can't come up with a fix ... I'll follow up with the EUI team.

Reply:
Looks like an issue with column header sorting on `EuiInMemoryTable` when the column `name` is a `ReactNode` rather than simple `text`, causing the click to sort in descending order not to be run. Please raise with the EUI team.

Reply:
Sorting on the anomaly score column is working for me with the latest edits.